clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -6,9 +6,7 @@ import java.util.Comparator
 
 import scala.io.Source
 import scala.sys.process.Process
-
-object IoUtil {
-
+trait IoUtilTrait {
```

Comment:
I'm not a huge fan of this name... I think it'd be better to leave it as `IoUtil`. You can still call the "real" object `IoUtil` if you want; the compiler will figure out which one you mean based on the usage context. Or you could rename the object to something more specific.

Reply:
I had originally done that, but IntelliJ yelled at me. Did it again, and realized it was a dismissable warning.